.github/workflows/sonar.yml

```diff
@@ -0,0 +1,34 @@
+name: Sonar
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '11' ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: print Java version
+        run: java -version
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: SonarSource
+        env:
+          SONAR_LOGIN: ${{ secrets.SONAR_LOGIN }}
```

Comment:
You set an environment variable called `SONAR_LOGIN` and then pass `$SONAR_SECRET`. I assume this is what causes your issue.